### ISSUE TITLE
is_baseline_system_11.2, is_baseline_system_12 Heating and Cooling 

### ISSUE THREAD

**Issue by JacksonJ-KC** [pnnl/ruleset-checking-tool#1384]
[](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_11.2.md?plain=1#L35)

[](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_12.md?plain=1#L37)

These baseline system functions are expecting terminal heating and terminal cooling to NOT exist to be considered as System 11 or 12, however this note in the schema indicates that the heating and cooling is supposed to be described at the terminal. 

> System configurations that typically are at the zone and include a compressor (such as packaged terminal air conditioning, packaged terminal heat pumps, window air conditioning units, and water loop heat pumps) should be reported in the schema using HeatingSystem and CoolingSystem. Systems that include gas or electric furnaces should be reported in the schema using HeatingSystem. **System configurations that are at the zone and only include fans and coils (such as four-pipe fan coil, two-pipe fan coil, radiant systems, baseboards, and chilled beams) should be reported in the schema using Terminal with the chilled water and hot water systems described in the cooling_source and heating_source data elements (and any other relevant Terminal Data elements)**. Evaporative cooling systems should be described in CoolingSystem. Passive diffusers with no coil or fan should be described in Terminal.

**Comment by claperle** [comment]
Recommend that we clarify the meaning of "At the zone" which was intended to mean non-ducted equipment located inside the zone.

**Comment by JacksonJ-KC** [comment]
In the case of 4-pipe and 2-pipe fan coils, does the way Heating/Cooling should be described change based on whether it is ducted?

**Comment by claperle** [comment]
Yes, I think so because if it is ducted then it is no longer a terminal
unit. This note is focused on equipment that people tend to think of as
terminal equipment.

*Christina LaPerle (she/her)* PE, BEMP, CPHC®, CEM, CBCP
Principal Engineer
________________
[image: image.png]
Phone 860.266.6220
We are always looking for motivated professionals to join our team. See our
career options here <https://karpmanconsulting.net/available-careers>!

**Comment by JacksonJ-KC** [comment]
There was a valid concern that `is_supply_ducted` would not be available because it may not be possible to populate from model inputs, however if we go with Christina's suggested solution it seems like the main distinction for determining if heating/cooling belongs in a HeatingSystem/CoolingSystem vs. in a Terminal - is whether it is ducted.

My suggestion is to clarify the bolded text above by saying:

> **Single-zone system configurations that** only include fans and coils (such as four-pipe fan coil, two-pipe fan coil, radiant systems, baseboards, and chilled beams) should be reported in the schema using Terminal with the chilled water and hot water systems described in the cooling_source and heating_source data elements (and any other relevant Terminal Data elements).

This will provide clear instructions to RPD generation developers and will allow predictable representation of systems that is not dependent on `is_supply_ducted`. It will also mean revisions to `is_baseline_system_11.2`, and `is_baseline_system_12` to allow/expect terminal heating/cooling

**Comment by claperle** [comment]
I am not sure about this. The note is meant for equipment that is considered Terminal equipment not all single zone systems. My clarification was to describe the meaning of "terminal equipment" and maybe some terminal equipment can be ducted so there is no useful definition that can be provided?  Jackson, have you looked at the other single zone AppG baseline system types to see if changes would need to be made based on your suggested change? Also, how would we distinguish between a ducted and non_ducted system using the approach you describe? I know this is circular logic because we are saying that the reason for this note is because is_supply_ducted may not be able to be populated but then are relying on the population of the coils at the terminal versus system level to be based on whether the equipment is terminal equipment which means that the tool would know whether it was ducted since terminal equipment is not ducted (right?). In other words, how else would a software tool know if the unit is a ducted single zone system like a system 12 or a terminal system like a FCU unless it knew details about the system such as whether it is ducted?
